packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -96,14 +95,6 @@ export const UnitRowRecoveryKey = () => {
           Restores your information when you forget your password.
         </p>
       </Localized>
-      <Localized id="rk-content-reset-data">
```

Comment:
I just did a quick search using the [rather unreliable] GitHub UI, but does anywhere else use this `rk-content-reset-data` string, or should we remove it from [the .ftl file](https://github.com/mozilla/fxa/blob/ee71759ff2c282a876c0e7e0ddb3a4e888aefc0d/packages/fxa-settings/src/components/UnitRowRecoveryKey/en-US.ftl#L13)? /cc @bcolsson @flodolo 

Looks like this SuMo content is still linked to from here, which is :+1::
https://github.com/mozilla/fxa/blob/ee71759ff2c282a876c0e7e0ddb3a4e888aefc0d/packages/fxa-settings/src/components/UnitRowTwoStepAuth/index.tsx#L166-L171

Reply:
@pdehaan Thank you for catching that.  I'll remove the message from the ftl file.